README.md

```diff
@@ -1 +1,70 @@
-# Math-Magician
\ No newline at end of file
+# Getting Started with Create React App
```

Comment:
- [ ] You need to edit your readme to follow the template provided by microverse.